package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@sentry/nextjs": "^6.17.4",
     "@sentry/node": "^6.19.5",
-    "axios": "^0.26.1",
+    "axios": "^1.2.1",
```

Comment:
Out of curiosity, was this version upgraded for any particular reason?

Reply:
TBH, I didn't realise it was installed. I thought I was adding it